test/test_um2netcdf.py

```diff
@@ -217,3 +220,139 @@ def test_cube_filtering_no_include_exclude(ua_plev_cube, heaviside_uv_cube):
+
+
+@pytest.fixture
+def max_cell_method():
+    return CellMethod("maximum")
+
+
+@pytest.fixture
+def min_cell_method():
+    return CellMethod("minimum")
+
+
+def test_rename_cube_var_name_simple(x_wind_cube, empty_um_var):
+    # NB: ignores cell methods functionality
+    assert x_wind_cube.var_name == "var_name"  # dummy initial value
+
+    for um_var in (None, empty_um_var):
+        um2nc.rename_cube_var_name(x_wind_cube, um_var, simple=True)
+        assert x_wind_cube.var_name == "fld_s00i002", f"Failed with um_var={um_var}"
+
+
+def test_rename_cube_var_rename_cell_methods_adds_max(x_wind_cube,
+                                                      empty_um_var,
+                                                      max_cell_method):
+    # ensure maximum cell methods add suffix to cube name
+    x_wind_cube.cell_methods = [max_cell_method]
+
+    for um_var in (None, empty_um_var):
+        um2nc.rename_cube_var_name(x_wind_cube, um_var, simple=True)
+        assert x_wind_cube.var_name == "fld_s00i002_max"
+
+
+def test_rename_cube_var_rename_cell_methods_adds_min(x_wind_cube,
+                                                      empty_um_var,
+                                                      min_cell_method):
+    # ensure maximum cell methods add suffix to cube name
+    x_wind_cube.cell_methods = [min_cell_method]
+
+    for um_var in (None, empty_um_var):
+        um2nc.rename_cube_var_name(x_wind_cube, um_var, simple=True)
+        assert x_wind_cube.var_name == "fld_s00i002_min"
+
+
+def test_rename_cube_var_name_unique(x_wind_cube):
+    # ensure um unique name given to cubes in non-simple mode
+    # NB: ignores cell methods functionality
+    unique = "unique_string_name"
+    um_unique = UMStash("", "", "", "", unique)
+    um2nc.rename_cube_var_name(x_wind_cube, um_unique, simple=False)
+    assert x_wind_cube.var_name == unique
+
+
+def test_rename_cube_standard_name_x_wind(x_wind_cube, empty_um_var):
+    # test cube wind renaming block only
+    # use empty um_var_empty_std to skip renaming logic
+    um2nc.rename_cube_standard_names(x_wind_cube, empty_um_var, verbose=False)
+    assert x_wind_cube.standard_name == "eastward_wind"
+
+
+def test_rename_cube_standard_name_y_wind(empty_um_var):
+    # test cube wind renaming block only
+    # use empty um_var_empty_std to skip renaming logic
+    m_cube = PartialCube("var_name", {'STASH': DummyStash(0, 3)}, "y_wind")
+    m_cube.cell_methods = []
+
+    um2nc.rename_cube_standard_names(m_cube, empty_um_var, verbose=False)
+    assert m_cube.standard_name == "northward_wind"
+
+
+def test_cube_um_standard_name_mismatch(x_wind_cube):
+    # ensure mismatching standard names between cube & um uses the um std name
+    um_var = UMStash("", "", "", "fake", "")
+    um2nc.rename_cube_standard_names(x_wind_cube, um_var, verbose=False)
+    assert x_wind_cube.standard_name == um_var.standard_name
+
+
+def test_test_cube_um_standard_name_mismatch_warn(x_wind_cube):
```

Comment:
Was `test_test_` intentional? 
```suggestion
def test_cube_um_standard_name_mismatch_warn(x_wind_cube):
```

Reply:
Ahh no, that's me being unco! It's fixed here: https://github.com/ACCESS-NRI/um2nc-standalone/pull/52/commits/9a560fe5f1e949f0489b1fb3440ebef5d2e0eac9.